_maps/map_files/Theseus/TGS_Theseus.dmm

```diff
@@ -878,6 +878,10 @@
 	dir = 9
 	},
 /area/mainship/living/officer_rnr)
+"acq" = (
+/obj/machinery/computer/camera_advanced/remote_fob,
+/turf/open/floor/mainship/mono,
+/area/mainship/hallways/hangar)
```

Comment:
Why are you changing the theseus?

Reply:
That was an accident, i don't know why I didn't see that change earlier?